internal/subscription/database/subscription_repository.go

```diff
@@ -116,10 +116,14 @@ func (r SubscriptionRepository) UpdateSubscription(sub *domain.Subscription) err
 	if err := stmt.Get(sub, args...); err != nil {
```

Comment:
I think that now you can change it to `stmt.Exec`

Reply:
https://github.com/sumelms/microservice-course/pull/28/commits/0ba63413eecc914244b9b0cab324948e93aecaef